parts/common/components.json

```diff
@@ -930,47 +895,14 @@
               },
               {
                 "renovateTag": "<DO_NOT_UPDATE>",
-                "latestVersion": "1.6.21",
-                "previousLatestVersion": "1.6.18"
-              }
-            ],
-            "downloadURL": "https://packages.aks.azure.com/azure-cni/v${version}/binaries/azure-vnet-cni-windows-amd64-v${version}.zip"
-          }
-        }
-      }
-    },
-    {
-      "name": "azure-vnet-cni-swift-windows",
-      "windowsDownloadLocation": "c:\\akse-cache\\win-vnet-cni\\",
-      "downloadURIs": {
-        "windows": {
-          "default": {
-            "versionsV2": [
-              {
-                "renovateTag": "<DO_NOT_UPDATE>",
-                "latestVersion": "1.4.59",
-                "previousLatestVersion": "1.4.58"
-              }
-            ],
-            "downloadURL": "https://packages.aks.azure.com/azure-cni/v${version}/binaries/azure-vnet-cni-swift-windows-amd64-v${version}.zip"
-          }
-        }
-      }
-    },
-    {
-      "name": "azure-vnet-cni-overlay-windows",
-      "windowsDownloadLocation": "c:\\akse-cache\\win-vnet-cni\\",
-      "downloadURIs": {
-        "windows": {
-          "default": {
-            "versionsV2": [
+                "latestVersion": "1.6.30"
+              },
               {
                 "renovateTag": "<DO_NOT_UPDATE>",
-                "latestVersion": "1.4.59",
-                "previousLatestVersion": "1.4.58"
+                "latestVersion": "1.7.0-0"
```

Comment:
some here, this is to download the tgz so 1.7.0-0 is not valid. i think these don't need to be updated past 1.5/1.6